ang/civicase/CaseDetails--tabs.html

```diff
@@ -5,6 +5,54 @@
       <span class="sr-only" ng-if="activeTab === tab.name">{{ ts('Current') }}</span>
     </a>
   </li>
+  <li class="pull-right" civicase-dropdown>
+    <div class="civicase__case-details__add-new-dropdown">
+      <button type="button" class="btn btn-primary" civicase-dropdown-toggle>
+        <i class="fa fa-plus-circle"></i>Add new<i class="fa fa-caret-down"></i>
+      </button>
+      <ul class="dropdown-menu">
+        <li><a class="crm-popup" ng-href="{{ 'civicrm/activity/email/add' | civicaseCrmUrl:{ action: 'add', caseid: item.id, reset: 1, cid: role.contact_id } }}"><i class="material-icons">email</i> {{ts('Create email')}}</a></li>
+        <li><a class="crm-popup" ng-href="{{ 'civicrm/activity/pdf/add' | civicaseCrmUrl:{ action: 'add', caseid: item.id, reset: 1, cid: role.contact_id, context: 'standalone' } }}"><i class="material-icons">picture_as_pdf</i> {{ts('Create PDF letter')}}</a></li>
+        <li class="divider"></li>
+        <li class="civicase__activity-dropdown" civicase-dropdown>
+          <a href civicase-dropdown-toggle>
+            {{ts('Add activity')}}
+            <i class="fa fa-caret-right pull-right"></i>
+          </a>
+          <ul class="dropdown-menu civicase__dropdown-menu--filters">
+            <li class="form-group has-feedback">
+              <input class="form-control" type="text" ng-model="activityTypeSearchText" />
+              <i class="fa fa-search form-control-feedback"></i>
```

Comment:
Material icons needs to be used.

Reply:
In the original design the search icon is defined using font-awesome. Not sure if all icons must be defined using material icons, if so we'll need to confirm this change with Jon